### ISSUE TITLE
props.hsl-colors.css hsl() syntax

### ISSUE THREAD

**Issue by RickCogley** [argyleink/open-props#345]
I downloaded props.hsl-colors.css to a vendor folder and included it via @import in my css after the main open props, but I could not get these props working, until I added hsl() and separated with commas as in this gist: 

https://gist.github.com/RickCogley/1c6855203e6dff0072a671a0149cacfa

Is the hsl() not needed or was this simply something I was doing incorrectly?

**Comment by argyleink** [comment]
what browser were you using? the older (deprecated) syntax uses commas, but going forward they arent needed.

here's a codepen i made which loads in the hsl colors and then uses one on the page bg https://codepen.io/argyleink/pen/JjBVgVb

**Comment by RickCogley** [comment]
Thanks! I was using the latest edge on latest macOS. I'll find the exact versions when I'm logged in.

**Comment by RickCogley** [comment]
It's Edge Version 109.0.1518.61 (Official build) (x86_64) on MacOS Ventura 13.1

**Comment by argyleink** [comment]
cool, just tested 109 Edge and the demo looks as expected. hope this helps unblock you!

**Comment by RickCogley** [comment]
Right, thanks! The codepen showed me what I needed to do. I was referencing like this: 

```
--color-background-1: var(--stone-2-hsl);
```

... but needed to be doing this: 

```
--color-background-1: hsl(var(--stone-2-hsl));
```

Once I added the `hsl()` around the var call, it started to work. I guess what happened is, I discovered I needed the `hsl()` function in a roundabout way, but had applied it on the file I'd downloaded into vendor as shown in my gist, rather than direct in my css.
